ci_default_post_tendrl.yml

```diff
@@ -15,8 +15,11 @@
 
     - debug: var=admin_password
 
-    - name: Copy usm.ini template
-      local_action: template src=./templates/usm.ini.j2 dest=./usm.ini
+    - name: Copy defaults.yaml template
+      local_action: template src=./templates/defaults.yaml.j2 dest=./defaults.yaml
```

Comment:
I added there `defaults.yaml` because of line:
```
password: {{ hostvars[groups['usm_server'][0]]['admin_password'].stdout }}
```
It seems that this attribute can be modified but if we hardcode password to *adminuser*, it can be pushed out.

Reply:
I think, the value `adminuser` is quite surely the correct one, so I think it can be hardcoded as default in the *usmqe-tests*. If it should be *temporarily*changed, it should be possible to update it in *usm.yaml*, right?

Reply:
Ok, I removed `defaults.yaml` template.